.github/pull_request_template.md

```diff
@@ -19,3 +19,4 @@ particularly focused on _why_ the work is necessary, not _what_ you did.
 * [ ] Passes all tests
 * [ ] Add comments such that someone without knowledge of the code could understand
 * [ ] Add Devel Docs in the `doc/` directory for any new code design
+* [ ] For external authors, check contributors' agreement and explict comment for ackowledgement.
```

Comment:
```suggestion
* [ ] You or your company has a signed contributor's agreement on file with Argonne
* [ ] For non-Argonne authors, request an explicit comment from your companies PR approval manager
```

Reply:
I'll make the change.

Do you mind I remove the checkbox for "Add Devel Docs in the doc/..."?

Reply:
Sure.